src/goFormat.ts

```diff
@@ -27,7 +27,7 @@ export class Formatter {
 			}
 
 			// Fix for https://github.com/Microsoft/vscode-go/issues/613 and https://github.com/Microsoft/vscode-go/issues/630
-			if (formatTool === 'goimports') {
+			if (formatTool === 'goimports' || formatTool === 'goreturns') {
```

Comment:
For people who haven't updated their `goreturns` this would result in an error of the form `flag provided but not defined: -srcdir`

In that case, we need to prompt the user to update their `goreturns` tool. An example of how we do this is here:
https://github.com/Microsoft/vscode-go/blob/0.6.70/src/goSymbol.ts#L89

Reply:
Update. But meet a error. Load the extension and edit a go file then save, the prompt for update show and disappear and show and disappear...

Reply:
Are you saying that the prompt shows up for each save? If yes, then its fine. If the user presses the `Close` button or presses the `Escape` key, then the prompt will no longer show up in the current session of VS Code.

Reply:
Nope, in my case, the prompt shows up and disappear quickly again and again.

Reply:
So the prompt appears and disappears even without you typing anything? If it happens only when you type, then do you have auto-save on?

Reply:
I have auto-save disabled, it happens on manually save.

Reply:
If you click on the Close button or escape key, does it still keep appearing and disappearing on each file save?